config/initializers/kafka.rb

```diff
@@ -13,7 +13,7 @@ class AsyncKafkaProducer
   def self.instance
     @@instance ||=
       begin
-        kafka = Kafka.new([Rails.application.secrets.kafka[:broker_host]],
+          kafka = Kafka.new(Rails.application.secrets.kafka[:broker_host].split(','),
```

Comment:
will this always need a split?

Reply:
Yes, the MSK always has at least 3 brokers. Also, it removed your casting of a string to a single-item list (since the API requires a list), that's taken care of by the split-on-a-string-with-no-comma